alns/accept/SimulatedAnnealing.py

```diff
@@ -55,6 +55,11 @@ def __init__(
                 "start_temperature < end_temperature not understood."
             )
 
+        method = method.lower()
```

Comment:
Question: why do we want this? 
Suggestion: if we keep this, then we should also add this to RecordToRecordTravel.

Reply:
I forgot we already have this, in `update.py` (which is called from almost everywhere). So I'm removing it here.